session.go

```diff
@@ -727,6 +727,11 @@ func (s *session) Execute(goCtx goctx.Context, sql string) (recordSets []ast.Rec
 			return nil, errors.Trace(err)
 		}
 	} else {
+		err = s.loadCommonGlobalVariablesIfNeeded()
```

Comment:
How about use prepared statement to run SQL?

Reply:
Fixed